esmvaltool/recipes/recipe_cvdp.yml

```diff
@@ -20,7 +20,28 @@ documentation:
     - c3s-magic
 
 datasets:
-  - {dataset: MPI-ESM-LR,  project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2002}
+  - {dataset: ACCESS1-0,  project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1850,  end_year: 2005}
```

Comment:
`mip` might be better specified with the variables instead of with the datasets.

Reply:
Addressed in b76ef63 ,
done